subprojects/performance/templates.gradle

```diff
@@ -351,7 +351,7 @@ tasks.register("k9AndroidBuild", RemoteProject) {
 
 tasks.register("largeAndroidBuild", RemoteProject) {
     remoteUri = 'https://github.com/gradle/perf-android-large.git'
-    branch = 'android-31'
+    branch = 'android-33'
```

Comment:
@lptr Are you OK with the upgrade of the test project to `android-33`? Note that this will be for all android performance tests.

Reply:
If they run, I think that should be fine. Am I missing something?